posthog/hogql/functions/mapping.py

```diff
@@ -578,6 +579,7 @@ class HogQLFunctionMeta:
     "count": HogQLFunctionMeta("count", 0, 1, aggregate=True),
     "COUNT": HogQLFunctionMeta("count", 0, 1, aggregate=True),
     "countIf": HogQLFunctionMeta("countIf", 1, 2, aggregate=True),
+    "countDistinctIf": HogQLFunctionMeta("countIf", 1, 2, aggregate=True),
```

Comment:
For consistency we should probably support all -Distinct functions https://github.com/PostHog/posthog/issues/20662.